	Exception(예외처리)
		프로그램이 실행 중에 에러가 발생하면 아무런 처리도 없이 강제 종료됨,
		프로그램을 강제로 종료가 되는걸 막기 위해서
		예외처리를 이용해서 실행하는 도중 에러가 발생하면
		해당 에러를 잡아서 처리를 한 후에
		남은 작업을 이어서 하게끔 처리하는 방법
		
	try : 작업 영역(실행할 코드), 이 영역에서는 Exception이 발생 할 수 있음.
	catch : try 영역에서 Exception이 발생 했을 때, 
			해당 Exception을 처리하는 영역
	finally : 
	throw : 
	throws : 